README.md

```diff
@@ -4,6 +4,8 @@ If you are unfamiliar with PerfView, there are [PerfView video tutorials](http:/
 As well as [Vance Morrison's blog](http://blogs.msdn.com/b/vancem/archive/tags/perfview) which also gives overview and getting 
 started information. 
 
+[![Build status](https://ci.appveyor.com/api/projects/status/fxtu3xa874whk2w0?svg=true)](https://ci.appveyor.com/project/sharwell/perfview)
```

Comment:
My experience is that with documentation you should err on the side of too much rather than too little (since the bad effect of too much is small and the bad effect of too little can be great).   I agree we should not put in information that quickly gets out of date.  I am talking about a sentence on this page, and a link to a new page with what can start as very small paragraph that tells you the value.     If you are averse to this, let me know and I will do it.  

On Security:  I mainly was interested in how trustworthy the output artifacts are (do we expect people to run the output from it, and if so, what risk are they taking?).